### ISSUE TITLE
Iterate text for media alt description

### ISSUE THREAD

**Issue by karmatosed** [WordPress/gutenberg#19068]
Currently, this pulls from the core so there is a trac ticket needs also iterating: https://core.trac.wordpress.org/ticket/48939. That said, we can explore here and solve both.

Right now, the description and link to an outside source are a little difficult to understand. What can we do to make this clearer?

![1](https://user-images.githubusercontent.com/253067/70638851-79c2b480-1c31-11ea-8631-2cf073c7d3d3.png)

**Comment by michelleweber** [comment]
What about something like:

Alt Text explains an image's purpose, giving context to visitors with vision impairments. It's recommended for all key images.

or, for a shorter version

Alt Text explains an image's purpose, giving context to visitors with vision impairments.

**Comment by karmatosed** [comment]
I sort of like the shorter version myself.

**Comment by carolynannewells** [comment]
Hello! Here from editorial. Were you looking for an alternative to Michelle's suggestion above? Her shorter version: Alt Text explains an image's purpose, giving context to visitors with vision impairments.

**Comment by carolynannewells** [comment]
Or: Provide a description of the image to improve accessibility for all visitors.

**Comment by priethor** [comment]
Hi there! This issue came up during the recent [Go/No-go demo on April 14th](https://make.wordpress.org/core/2021/04/15/full-site-editing-go-no-go-april-14-2021/), with the following suggestions:
- Expand the description to explain why adding an alternative text benefits SEO.
- Link to a wordpress.org resource as further explanation instead of an external resource.

**Comment by joedolson** [comment]
In discussing the core ticket (https://core.trac.wordpress.org/ticket/48939), we agreed that we do not want to use a WordPress resource for the alt text reference link.

There are several reasons, but the most significant are that 1) it creates a long-term permanent burden on the a11y and docs teams to maintain the resource and 2) it doesn't effectively reflect that these are global standards of accessibility, rather than an arbitrary recommendation from the WordPress project.

Conversation in Slack: https://wordpress.slack.com/archives/C02SX62S6/p1636047226109000

**Comment by karmatosed** [comment]
I am going to close this ticket as both the recommendations seem to lean there and that feels the next best step.
